src/lib/util.ts

```diff
@@ -116,3 +117,9 @@ export const gleeMessageToFunctionEvent = (message: GleeMessage, glee:Glee): Gle
     glee,
   } as GleeFunctionEvent
 }
+
+export const isRemoteServer = (parsedAsyncAPI: AsyncAPIDocument, serverName: string): boolean => {
+  const remoteServer = parsedAsyncAPI.extension('x-remoteServers')
+  if (remoteServer && remoteServer.includes(serverName)) return true
+  return false
```

Comment:
You can simplify this further:

```suggestion
  const remoteServers = parsedAsyncAPI.extension('x-remoteServers')
  return remoteServers && remoteServers.includes(serverName)
```

Also, renamed the variable to a plural name since it's returning an array of servers, not just one.